strawberryfields/backends/fockbackend/ops.py

```diff
@@ -500,3 +498,68 @@ def hermiteVals(q_mag, num_bins, m_omega_over_hbar, trunc):
         Hvals[i] = 2 * x * Hvals[i - 1] - 2 * (i - 1) * Hvals[i - 2]
 
     return q_tensor, Hvals
+
+
+def gkp_displacements(t, k, epsilon):
+    """
+    Helper function to generate the displacements parameters associated with the teeth of
+    GKP computational basis state k.
+    """
+    return np.sqrt(0.5 * np.pi) * (2 * t + k) / np.cosh(epsilon)
+
+
+def gkp_coeffs(t, k, epsilon):
+    """
+    Helper function to generate the coefficient parameters associated with the teeth of
+    GKP computational basis state k.
+    """
+    return np.exp(-0.5 * np.pi * np.tanh(epsilon) * (k + 2 * t) ** 2)
+
+
+@functools.lru_cache()
+def squaregkpBasisState(i, epsilon, ampl_cutoff, cutoff):
```

Comment:
functions would follow a name with underscores in between words rather than camelcase
```suggestion
def square_gkp_basis_state(i, epsilon, ampl_cutoff, cutoff):
```

Reply:
errr.... so I get your point that this is the ocnvention, except that this convention is nowhere followed in this file 
![image](https://user-images.githubusercontent.com/991946/110872472-e0ec7000-829d-11eb-8070-15ee91c470ab.png)